dissect/target/plugins/os/windows/defender/mplog.py

```diff
@@ -89,7 +89,7 @@
         ("varint", "pid"),
         ("string", "sigseq"),
         ("varint", "send_memory_scan_report"),
-        ("varint", "source"),
+        ("varint", "source_int"),
```

Comment:
I'm not a big fan of these `_int` suffixes, surely there's a better alternative for where you currently do this. Maybe `source_id` for this specific instance?